2020.05.04 John Dey jfdey@fredhutch.org - Replace C header with CPP header
diff -ruN trinityrnaseq-v2.12.0.orig/trinity-plugins/bamsifter/sift_bam_max_cov.cpp trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/sift_bam_max_cov.cpp
--- trinityrnaseq-v2.12.0.orig/trinity-plugins/bamsifter/sift_bam_max_cov.cpp	2021-03-04 17:22:55.000000000 -0800
+++ trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/sift_bam_max_cov.cpp	2021-05-04 12:18:52.078419000 -0700
@@ -2,7 +2,8 @@
 #include <stdio.h>
 // #include <unistd.h>
 #include <stdlib.h>
-#include <string.h>
+#include <string>
+#include <cstring>
 #include <getopt.h>
 #include <stdint.h>
 #include <limits.h>
